CHANGELOG.rst

```diff
@@ -1,7 +1,22 @@
 Changelog
 =========
 
+0.2.0
+`````
+
+Version 0.2.0 updates python-bsonjs's vendored copy of libbson to 1.6.5
+http://mongoc.org/libbson/1.6.2/ and fixes an error in our Linux builds.
```

Comment:
Good point updated to say previous releases for Python 3.3/3.4 on Linux do not work. I also added some JSON specific info from the libbson 1.4/1.5/1.6 release notes.